packaging/radicle-daemon.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
```

Comment:
Should we not express that this service `Wants` the radicle-ipfs sefvice?

Reply:
We define `Requires=radicle-ipfs.service` below. This is a stronger version of [`Wants=`](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#Wants=)